.vscode/settings.json

```diff
@@ -1,3 +1,86 @@
 {
-  "cSpell.words": ["collaboratives"]
+  "psi-header.config": {
```

Comment:
The `psi-header` configuration is standard, used throughout all repositories.

... and now I realize that I should add file headers to relevant files.